Makefile.am

```diff
@@ -25,7 +25,6 @@ libfastjsoninclude_HEADERS = \
 
 libfastjson_la_LDFLAGS = -version-info 3:0:0 -no-undefined @JSON_BSYMBOLIC_LDFLAGS@ \
 	-export-symbols-regex '^fjson_.*'
-libfastjson_la_CPPFLAGS = -Werror
```

Comment:
2016-04-07 16:53 GMT+02:00 Michael Biebl <notifications@github.com>:

> In Makefile.am
> <https://github.com/rsyslog/libfastjson/pull/47#discussion_r58885088>:
>
> > @@ -25,7 +25,6 @@ libfastjsoninclude_HEADERS = \
> >
> >  libfastjson_la_LDFLAGS = -version-info 3:0:0 -no-undefined @JSON_BSYMBOLIC_LDFLAGS@ \
> >  	-export-symbols-regex '^fjson_.*'
> > -libfastjson_la_CPPFLAGS = -Werror
>
> Ok, will prep a separate PR for this once this PR has been merged.
> Do you prefer if a copy of ax_compiler_flags.m4 is included in m4 or do we
> rely on autoconf-archive being installed for those who wish to build from
> git?
>
What would you recommend? I think it should be as painless as possible for
casual developers, but on the other hand a properly maintained version from
the archive makes me feel good...

Rainer